lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -36,11 +36,12 @@ export class UvProcessor implements PyProjectProcessor {
     // Skip sources that are either not yet handled by Renovate (e.g. git), or do not make sense to handle (e.g. path).
     if (uv.sources) {
       for (const dep of deps) {
-        if (!dep.depName) {
+        // istanbul ignore if
+        if (!dep.packageName) {
```

Comment:
Not a case we can test with the new test location, and adding a test just for that in `uv.spec.ts` seems a bit overkill.